frontend/public/style/_common.scss

```diff
@@ -174,6 +174,9 @@
   dd {
     font-size: 14px;
     @include co-break-word;
+    .btn {
+      font-size: inherit;
```

Comment:
Do we know if this causes problems elsewhere? I think we do have buttons inside `dd` elements (for instance, copy to clipboard).

Reply:
I actually meant to make this `btn.btn-link` because the link styled buttons should have the same size font as normal text. I think it would be okay for the other buttons to keep their original styling, but it looks weird to have a 13px link mixed in with 14px text.

Reply:
Also, the full selector for this would be `.co-m-pane__details dd .btn.btn-link`.  I think that this would be specific enough to be safe.

Reply:
I'd really like to avoid special casing such as this.  The real issue is we shouldn't be changing the size of `dd`.  I recommend *not* adding this override and having @sg00dwin look at removing line 175.

Reply:
Sounds good. I'll just take this override out.